AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -253,6 +253,20 @@ - (void)deleteLogsWithBatchId:(NSString *)batchId groupId:(NSString *)groupId {
 
 #pragma mark - DB selection
 
+- (NSDate *)getOldestLogTime:(NSString *)groupId {
+  NSMutableString *query = [NSMutableString stringWithFormat:@"SELECT MIN(\"%@\") FROM \"%@\" WHERE \"%@\" = '%@'", kMSTimestampColumnName,
+                                                             kMSLogTableName, kMSGroupIdColumnName, groupId];
+  NSArray<NSArray *> *entries = [self executeSelectionQuery:query];
+  NSTimeInterval timestamp = 0;
```

Comment:
Nit: I'd move declaration inside `if` block, and outside ([line 267 ](https://github.com/microsoft/appcenter-sdk-apple/pull/1630/files#diff-8faf03edd2017a6e6cc1e7898cf0ea74R267)) just return
```return [NSDate dateWithTimeIntervalSince1970:timestamp];```